src/Uno.UI/UI/Xaml/Controls/TextBlock/TextBlockMeasureCache.cs

```diff
@@ -36,28 +32,38 @@ internal partial class TextBlockMeasureCache
 		/// <returns>An optional <see cref="Size"/> if found.</returns>
```

Comment:
In the `_entries` you could use the comparer to `MeasureKey.Comparer.Insatnce` to avoid to create a new instance for nothing :)

Reply:
That's something in the Jérôme's code I didn't look/updated.